homework/tests/homework.spec.js

```diff
@@ -0,0 +1,230 @@
+      await page.locator('#calculateButton').click();
+  
+      var error = await page.textContent('#errorMsgField');
+      await expect(error).toContain('Divide by zero error!');
+    });
+
+    test('Adding 2.5 and 3 results in 5.5', async ({ page }) => {
+      await page.goto('https://testsheepnz.github.io/BasicCalculator');
+      await page.selectOption('#selectBuild', { label: version});
+      await page.locator('#number1Field').type('2.5');
+      await page.locator('#number2Field').type('3');
+      await page.selectOption('#selectOperationDropdown', {label: 'Add'});
+      await page.locator('#calculateButton').click();
+  
+      await expect(page.locator('#numberAnswerField')).toHaveValue('5.5');
+    });
+
+    test('Concatenating 2.5 and 3 results in 2.53', async ({ page }) => {
+      await page.goto('https://testsheepnz.github.io/BasicCalculator');
+      await page.selectOption('#selectBuild', { label: version});
+      await page.locator('#number1Field').type('2.5');
+      await page.locator('#number2Field').type('3');
+      await page.selectOption('#selectOperationDropdown', {label: 'Concatenate'});
+      await page.locator('#calculateButton').click();
+  
+      await expect(page.locator('#numberAnswerField')).toHaveValue('2.53');
+    });
+
+    test('Subtract 2.5 from 3 results in 0.5', async ({ page }) => {
+      await page.goto('https://testsheepnz.github.io/BasicCalculator');
+      await page.selectOption('#selectBuild', { label: version});
+      await page.locator('#number1Field').type('3');
+      await page.locator('#number2Field').type('2.5');
+      await page.selectOption('#selectOperationDropdown', {label: 'Subtract'});
+      await page.locator('#calculateButton').click();
+  
+      await expect(page.locator('#numberAnswerField')).toHaveValue('0.5');
+    });
+
+    test('Multiply 2.5 by 3 results in 7.5', async ({ page }) => {
+      await page.goto('https://testsheepnz.github.io/BasicCalculator');
+      await page.selectOption('#selectBuild', { label: version});
+      await page.locator('#number1Field').type('3');
+      await page.locator('#number2Field').type('2.5');
+      await page.selectOption('#selectOperationDropdown', {label: 'Multiply'});
+      await page.locator('#calculateButton').click();
+  
+      await expect(page.locator('#numberAnswerField')).toHaveValue('7.5');
+    });
+
+    test('Dividing 6 from 4 results in 1.5', async ({ page }) => {
+      await page.goto('https://testsheepnz.github.io/BasicCalculator');
+      await page.selectOption('#selectBuild', { label: version});
+      await page.locator('#number1Field').type('6');
+      await page.locator('#number2Field').type('4');
+      await page.selectOption('#selectOperationDropdown', {label: 'Divide'});
+      await page.locator('#calculateButton').click();
+  
+      await expect(page.locator('#numberAnswerField')).toHaveValue('1.5');
+    });
+
+    test('Adding 4 and k results in an error message', async ({ page }) => {
```

Comment:
Optional.
Please avoid specific values in test name.
Rather use their types -> `Adding integer and string results in error message`

In this case, if we change what values we enter in the test, we do not need to revisit the test name, so they would match.